integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledClearAlarmRegister.feature

```diff
@@ -0,0 +1,39 @@
+@SmartMetering @Platform
+Feature: SmartMetering Bundle - ClearAlarmRegister
+  As a grid operator 
+  I want to be able to clear the alarm register from a meter via a bundle request
+
+  Scenario: Clear alarm register SMR 5.1
+    Given a dlms device
+      | DeviceIdentification     | TEST1024000000002 |
+      | DeviceType               | SMART_METER_E     |
+      | SelectiveAccessSupported | true              |
+      | Protocol                 | SMR               |
+      | ProtocolVersion          | 5.1               |
+    And device "TEST1024000000002" has alarm register "1" with some value
+    And device "TEST1024000000002" has alarm register "2" with some value
```

Comment:
This is an SMR5.1 meter. In other cucumber tests we used `TEST1028000000001` as identification for SMR5.1 meters.

Reply:
Changed to 1028